src/client/components/MyProfile/YourUploadedCVs/CVList.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './YourUploadedCVs.css';
+
+export function CVList({ title, date }) {
```

Comment:
Please fix these warnings

Reply:
I really do not know which warnings do you mean. Do you mean props types?

Reply:
If you mean warnings about props types I made them. However they were not mentioned obligatory in the project :)